akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -0,0 +1,465 @@
+.. _remoting-artery-scala:
+
+##########################
+Remoting (codename Artery)
+##########################
+
+.. note::
+
+  This page describes the experimental remoting subsystem, codenamed *Artery* that will eventually replace the
+  old remoting implementation. For the current stable remoting system please refer to :ref:`_remoting-scala`.
+
+Remoting enables Actor systems on different hosts or JVMs to communicate with each other. By enabling remoting
+the system will start listening on a provided network address and also gains the ability to connect to other
+systems through the network. From the application's perspective there is no API difference between local or remote
+systems, :class:`ActorRef` instances that point to remote systems look exactly the same as local ones: they can be
+sent messages to, watched, etc.
+Every :class:`ActorRef` contains hostname and port information and can be passed around even on the network. This means
+that on a network every :class:`ActorRef` is a unique identifier of an actor on that network.
+
+Remoting is not a server-client technology. All systems using remoting can contact any other system on the network
+if they possess an :class:`ActorRef` pointing to those system. This means that every system that is remoting enabled
+acts as a "server" to which arbitrary systems on the same network can connect to.
+
+What is new in Artery
+---------------------
+
+Artery is a reimplementation of the old remoting module aimed at improving performance and stability. It is mostly
+backwards compatible with the old implementation and it is a drop-in replacement in many cases. Main features
+of Artery compared to the previous implementation:
+
+* Based on `Aeron <https://github.com/real-logic/Aeron>`_ instead of TCP
+* Focused on high-throughput, low-latency communication
+* Mostly allocation-free operation
```

Comment:
"improved cluster stability by using isolated streams for control messages, this means less false failure detection in case of heavy message traffic" ?